index.html

```diff
@@ -7,48 +7,167 @@
-          </li>
-          <li>
-            <a class="link" href="https://twitter.com/AjrassTajemouti" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/twitter.svg" alt="Twitter">
-            </a>
-          </li>
-          <li>
-            <a class="link" href="https://medium.com/@tajemouti/" rel="noopener noreferrer" target="_blank">
-              <img src="./svg/medium.svg" alt="Medium">
-            </a>
-          </li>
+    <section class="headline background">
+      <h1 class="hello">Hey There.<br>I'm Ajrass.</h1>
+      <h2 class="iam">I'm a Software Developer</h2>
+      <p class="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+      <ul class="social">
+        <li>
+          <a class="link" href="https://github.com/tajemouti" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconGitHub.svg" alt="GitHub">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://www.linkedin.com/in/ajrass" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/Icon Linkedin.svg" alt="LinkedIn">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://wellfound.com/u/ajrass-tajemouti" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconAngelist.svg" alt="WellFound">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://twitter.com/AjrassTajemouti" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconTwitter.svg" alt="Twitter">
+          </a>
+        </li>
+        <li>
+          <a class="link" href="https://medium.com/@tajemouti/" rel="noopener noreferrer" target="_blank">
+            <img src="default/SVG/IconMedium.svg" alt="Medium">
+          </a>
+        </li>
+      </ul>
+    </section>
+    
+    <section class="work">
+      <h1 class="title">My Recent Works</h1>
+      <div class="frame">
+        
+        <hr class="line1">
+        <div class="prointro">
+          <img class="placeholder" src="default/PNG/img-placeholder.png" alt="imageholder">
+          <div>
+            <h2 class="stories">Multi-Post Stories</h2>
+            <p class="stories-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="lang">
+              <li class="lang-list">JavaScript</li>
+              <li class="lang-list">HTML</li>
+              <li class="lang-list">CSS</li>
+            </ul>
+            <button class="button" type="button">
+              <a href="#" target="_blank">See project</a>
+            </button>
+          </div>
+        </div>
```

Comment:
According to the Figma design, I suggest you make sure you stick to the design as much as possible by using the right margin and padding.

> ## The Figma Design
>
><img width="187" alt="Screenshot 2023-06-03 at 6 50 35 PM" src="https://github.com/tajemouti/portfolio/assets/91237525/d6f6f43f-0acd-40f9-9a52-205356f1d2a8">
>
> ## Your Design
>
><img width="372" alt="Screenshot 2023-06-03 at 6 48 49 PM" src="https://github.com/tajemouti/portfolio/assets/91237525/1fcf229d-3adc-4754-ac97-05d3ae0551b8">
><img width="375" alt="Screenshot 2023-06-03 at 6 49 42 PM" src="https://github.com/tajemouti/portfolio/assets/91237525/a9d330cc-0c86-4a10-ada4-a82efd012976">